sky/jobs/state.py

```diff
@@ -462,18 +462,25 @@ def set_failed(
 
 
 def set_cancelling(job_id: int, callback_func: CallbackType):
-    """Set tasks in the job as cancelling, if they are in non-terminal states.
+    """Set tasks in the job as cancelling, if they are not running and in
+    non-terminal states.
 
     task_id is not needed, because we expect the job should be cancelled
     as a whole, and we should not cancel a single task.
     """
+
+    # TODO: https://github.com/skypilot-org/skypilot/issues/4195 - Current
```

Comment:
```suggestion
    # TODO(andy): https://github.com/skypilot-org/skypilot/issues/4195 - Current
```

Reply:
Seems that the username might not be needed in the updated TODO style? Check https://google.github.io/styleguide/pyguide.html#312-todo-comments.

Reply:
It will help us to keep track of those TODOs ;) Pls check https://github.com/skypilot-org/skypilot/blob/master/CONTRIBUTING.md#some-general-engineering-practice-suggestions

Reply:
Got it!